cuda/preconditioner/jacobi_kernels.cu

```diff
@@ -75,7 +76,7 @@ size_type find_natural_blocks(std::shared_ptr<const CudaExecutor> exec,
 
     Array<bool> matching_next_row(exec, mtx->get_size()[0] - 1);
 
-    const dim3 block_size(default_block_size, 1, 1);
+    const dim3 block_size(config::warp_size, 1, 1);
```

Comment:
It is caused by confusing name.
`default_block_size` is the number of warp not the block size.
It only affects `initialize_precisions` now.
Could you also change the name like `default_num_warp` or something else?

Reply:
Good point, I renamed the variable.